packages/ui/src/components/markdown.tsx

```diff
@@ -73,6 +73,27 @@ export function Markdown({ children, className }: { children: string; className?
               style={syntaxHighlighterTheme} // theme
               wrapLines
               wrapLongLines
+              customStyle={{ fontSize: 'inherit' }}
+              codeTagProps={{
+                style: {
+                  // Default styles
+                  fontFamily:
+                    'Menlo, Monaco, Consolas, "Andale Mono", "Ubuntu Mono", "Courier New", monospace',
+                  color: 'rgb(212, 212, 212)',
+                  textShadow: 'none',
+                  direction: 'ltr',
+                  textAlign: 'left',
+                  whiteSpace: 'pre',
+                  wordSpacing: 'normal',
+                  wordBreak: 'normal',
+                  tabSize: 4,
+                  hyphens: 'none',
```

Comment:
Is there a specific reason for this?

Reply:
Added it because the code blocks had an inline fontsize of 13px

Reply:
If you don't include this I believe everything should be working as normal. I tried it locally without the added default styles in here.

Reply:
Everything else except the font family remained the same. Although the fonts look similar
<img width="470" alt="Screenshot 2023-10-24 at 09 31 29" src="https://github.com/typehero/typehero/assets/46888096/563ab394-53b5-4152-946a-334145d8a420">

Reply:
I think that's fine though, isn't that what the font was before adding this? For example, if you remove the entire `customStyle` and `codeTagProps`, are those the fonts you see as well?

Just trying to make sure we don't have redundant default styles here if they're already included (except for fontSize and lineHeight)!

Reply:
Otherwise, if the font actually changes to something else when you introduce the `customStyle` and `codeTagProps` then I think it's okay to leave the `fontFamily` as you have it and remove everything else!

Reply:
Oki, then it looks like it is resetting to some other fonts without it. I think how you included it is good to go then. I think it looks good! I'll let Chris take a look in the morning and he can merge if he thinks it's okay as well!